site/src/components/navigation/navigation-container.jsx

```diff
@@ -1,12 +1,31 @@
-import { useStore } from '@nanostores/react'
-import React from 'react'
-import { isMenuOpen } from '../../state/menuState'
+import { useSignal } from '@preact/signals'
```

Comment:
Should probably just swap this for a useState no need to bring in signals just for this.